rpm/0016-Enable-external-window-usage.patch

```diff
@@ -115,6 +115,7 @@ index c12d381..fd30aa1 100644
  
 +  if (gfxPrefs::UseExternalWindow()) {
```

Comment:
When tampering this one once again... Could you now remove this unnecessary if-block, line 110 guarantees that this is true.